css/style.css

```diff
@@ -0,0 +1,72 @@
+body {
+  font-family: Roboto;
+  font-weight: 900;
+}
+
+p {
+  letter-spacing: 2.4px;
+}
+
+.container {
+  margin: 0 auto;
+  text-align: center;
+}
+
+header > .container {
+  padding: 30px 0;
+}
+
+.item {
+  text-transform: uppercase;
+  padding: 0 50px;
+  text-decoration: none;
+  font-size: 18px;
+  letter-spacing: 2.7px;
+  color: #16a085;
+}
+
+.heading {
+  text-transform: uppercase;
+  font-size: 22px;
+  letter-spacing: 3.3px;
+  margin: 59px auto 34px auto;
+}
+
+.m-top {
+  margin-top: 25px;
+}
+
+.image {
+  height: 790px;
+  width: 100%;
+}
+
+.inner-container {
+  line-height: 26px;
+  padding: 0 102px;
+  margin: 0 auto;
+}
+
+.main {
+  background-color: #f5f6f8;
+}
+
+.content {
+  padding: 45px 100px 45px 0px;
+}
+
+footer > .container {
```

Comment:
plz don't rewrite styles with the current selector, each selector should have unique styles